heron/common/src/cpp/network/client.h

```diff
@@ -303,7 +274,6 @@ class Client : public BaseClient {
   std::unordered_map<REQID, std::pair<sp_string, void*> > context_map_;
 
   typedef std::function<void(IncomingPacket*)> handler;
-  std::unordered_map<std::string, handler> requestHandlers;
```

Comment:
We can always add it back if we need it in the future, so I don't worry about it for now.